impacthub/build.yml

```diff
@@ -21,4 +21,4 @@ services:
     env_file:
       - ../.env
     volumes:
-      - ./node-data:/root/.ixod
+      - ./node-data:/root/.ixo
```

Comment:
Is `.ixo` the correct directory? We'll need to update the PROJECT_DIR arg if so

Reply:
ixod is the correct directory. It was .ixo in the docker-compose.yml mount, I thought that's the correct source of truth. I confirmed it by running the node locally now.